Integrations/Integration-Wildfire.yml

```diff
@@ -0,0 +1,261 @@
+          mdResults += sep.join('|') + '\n';
+          t.forEach(function(entry){
+              var vals = [];
+              headers.forEach(function(h){
+                  vals.push(!!entry[h] ? formatCell(entry[h]) : ' ');
+              });
+              mdResults += vals.join('|') + '\n';
+          });
+      } else{
+          mdResults += 'No data returned\n';
+      }
+      return mdResults;
+    };
+
+    var sendRequest = function(url, body, headers) {
+        url.replace(/[\/]+$/, '');
+        var res = http(
+            url,
+            {
+                Method: 'POST',
+                Body: body,
+                Headers: headers
+            }
+        );
+        if (res.StatusCode !=200) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode +' '+errorDictReport[res.StatusCode]+ '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+        return res.Body;
+    };
+
+    var sendMultipartRequest = function(url, headers, file, apikey){
+        url.replace(/[\/]+$/, '');
+        var res = httpMultipart(
+          url,
+          file,
+          {
+            Method: 'POST',
+            Headers: headers,
+          },
+          {
+            'apikey': apikey,
+            'file' : file
+          },
+          params.insecure
+        );
+
+        if (res.StatusCode !=200) {
+          throw 'Request Failed.\nStatus code: ' + res.StatusCode +' '+errorDictUpload[res.StatusCode]+ '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+        return JSON.parse(x2j(res.Body));
+    };
+
+    var headers = {'Content-Type': ['application/x-www-form-urlencoded']};
+
+
+    /*MD table with title "Wildfire Report", starting at 'wildfire'
+    In content, File Entity object with the relevant field from the integration.
+    a. if file is malicious, context name is File/Malicious else File
+    */
+
+    //var contextFunc = function(body){}
+    var resToMD = function(body, title) {
+        return tblToMd(title, body, Object.keys(body));
+    };
+
+    var parseFile = function(body) {
+        var context = {};
+        context.File = [];
+        var ep = {};
+
+        ep.Type = body.filetype;
+        ep.MD5 = body.md5;
+        ep.SHA1 = body.sha1;
+        ep.SHA256 = body.sha256;
+        ep.Size = body.size;
+        if(body.filename){
+            ep.Name = body.filename;
+        }
+
+        context.File.push(ep);
+
+        if(body.malware == 'yes'){
+            context['File/Malicious'] = [];
+            context['File/Malicious'].push(ep);
+        }
+        return context;
+    };
+
+
+    // The command input arg holds the command sent from the user.
+    switch (command) {
+      // This is the call made when pressing the integration test button.
+      case 'test-module':
+          var res = http(
+              url,
+              {
+                  Method: 'POST',
+                  Body: 'apikey=37ddffe5490c01835b6645fb6ecfe9c9&format=xml&hash=7f638f13d0797ef9b1a393808dc93b94',
```

Comment:
A couple of things.
1. It should be the user API key in order to test his env.
2. Our apikey shouldn't be available on this public repo...